scipy/optimize/optimize.py

```diff
@@ -2012,7 +2012,8 @@ def _minimize_scalar_brent(func, brack=None, args=(),
                           success=nit < maxiter)
 
 
-def golden(func, args=(), brack=None, tol=_epsilon, full_output=0):
+def golden(func, args=(), brack=None, tol=_epsilon,
+           maxiter=500, full_output=0):
```

Comment:
New keyword arguments should be added after existing ones to preserve backwards compatibility.